src/Constants.maccatalyst.cs.in

```diff
@@ -9,5 +9,8 @@ namespace ObjCRuntime {
 
 		// iOS 9.0
 		public const string libcompressionLibrary = "/usr/lib/libcompression.dylib";
+
+		// xcode 13
+		public const string ApplicationServicesCoreGraphicsLibrary = "/System/Library/Frameworks/ApplicationServices.framework/Frameworks/CoreGraphics.framework/CoreGraphics";
```

Comment:
where is it used ?

https://developer.apple.com/documentation/colorsync/colorsyncprofileref?changes=_4&language=objc _hints_ to it, but no DllImport uses that constant

Reply:
It is used in src/CoreGraphics/CGEvent.cs https://github.com/xamarin/xamarin-macios/pull/12589/files#diff-c22d9304f9f0fa2e7659f5a3ec3e9a4e9d5fac1ab8375bf81bd5a26583edb7c9L27 which was added for maccatalyst.

Reply:
I don't see the constant string `ApplicationServicesCoreGraphicsLibrary` used there ?

Reply:
Here: https://github.com/xamarin/xamarin-macios/pull/12589/files#diff-50ac05c3df9f3ac7e627565014bf06b0567b9a5c28d5ec78b4003b9903ff7206R77

Reply:
Sometimes github links to a line number of a file in a PR SUCK, better link: https://github.com/xamarin/xamarin-macios/blob/main/src/CoreGraphics/CGEvent.cs#L80
```
[DllImport (Constants.ApplicationServicesCoreGraphicsLibrary)]
		extern static IntPtr CGEventCreateMouseEvent(IntPtr source, CGEventType mouseType, CGPoint mouseCursorPosition, CGMouseButton mouseButton);
			
		public CGEvent (CGEventSource source, CGEventType mouseType, CGPoint mouseCursorPosition, CGMouseButton mouseButton)
		{
			handle = CGEventCreateMouseEvent (source == null ? IntPtr.Zero : source.Handle, mouseType, mouseCursorPosition, mouseButton);
		}
```

Reply:
ok, I see it now 👓 thanks!